src/main/java/io/github/jhipster/domain/util/JSR310DateConverters.java

```diff
@@ -82,6 +82,28 @@ public ZonedDateTime convert(Date source) {
         }
     }
 
+    public static class StringToZonedDateTimeConverter implements Converter<String, ZonedDateTime> {
+        public static final StringToZonedDateTimeConverter INSTANCE = new StringToZonedDateTimeConverter();
+
+        private StringToZonedDateTimeConverter() {
+        }
+
+        public ZonedDateTime convert(String source) {
```

Comment:
Shouldn't it be `public static ZonedDateTime convert(String source)`?

Reply:
No, it should not.